backend/src/StdLibDarkInternal/Helpers/ProgramTypesToDarkTypes.fs

```diff
@@ -29,6 +40,14 @@ module Sign =
     | Positive -> DEnum(stdlibTyp [] "Sign" 0, "Positive", [])
     | Negative -> DEnum(stdlibTyp [] "Sign" 0, "Negative", [])
```

Comment:
I think common.dark should move to the new LanguageTools namespace as well - I think the need for `stdlibTyp` will disappear then